workflow/scripts/build_existing_chp_de.py

```diff
@@ -49,18 +43,32 @@ def clean_data(combustion, biomass, geodata, opts):
         "Inbetriebnahmedatum": "DateIn",
         "DatumEndgueltigeStilllegung": "DateOut",
         "Postleitzahl": "Postleitzahl",
-        "Breitengrad": "lat",
-        "Laengengrad": "lon",
+        "IndustryStatus": "Industry",
     }
     CHP_sel = CHP_raw[rename_columns.keys()].rename(columns=rename_columns)
-
-    # set missing information to match the powerplant data format
-    CHP_sel[["Set", "Country", "Efficiency"]] = ["CHP", "DE", ""]
-
+    
     # change date format
     CHP_sel.DateIn = CHP_sel.DateIn.str[:4].astype(float)
     CHP_sel.DateOut = CHP_sel.DateOut.str[:4].astype(float)
 
+    # delete duplicates - compromise only when KwkMastrNummer and Bruttoleistung is the same
+    strategies = {
+        "Name": "first",
+        "Fueltype": "first",
+        "Technology": "first",
+        "Capacity": "mean",
+        "Capacity_thermal": "mean",        
```

Comment:
```suggestion
        "Capacity": "mean", # dataset duplicates full KWK capacity for each block
        "Capacity_thermal": "mean", # dataset duplicates full KWK capacity for each block
```
This might be confusing to people who don't know the background. Let's add a comment. The natural thing would be to sum.